src/components/Navigation.tsx

```diff
@@ -8,9 +8,9 @@ import ChatIcon from "./icons/ChatIcon";
 import ProfileIcon from "./icons/ProfileIcon";
 import CalendarMonthIcon from "@mui/icons-material/CalendarMonth";
 
-export default function Navigation() {
+const Navigation = () => {
```

Comment:
```suggestion
export const Navigation = () => {
```

if there a reason why we do default exports? I find named one are usually best and less error prone.

Reply:
actually I had meant to refactor this like I did in other files. Thanks for pointing it out!